test/rules/no-missing-di.js

```diff
@@ -0,0 +1,72 @@
+import RuleTester from '../../lib/testers/rule-tester';
+
+import rule from '../../rules/no-missing-di';
+
+import {
+  readModdle
+} from '../../lib/testers/helper';
+
+
+RuleTester.verify('no-missing-di', rule, {
+  valid: [
+    {
+      moddleElement: readModdle(__dirname + '/no-missing-di/valid.bpmn')
+    },
+    {
+      moddleElement: readModdle(__dirname + '/no-missing-di/valid-sub-processes.bpmn')
+    },
+    {
+      moddleElement: readModdle(__dirname + '/no-missing-di/valid-data-object.bpmn')
+    },
+    {
+      moddleElement: readModdle(__dirname + '/no-missing-di/valid-error.bpmn')
+    },
+    {
+      moddleElement: readModdle(__dirname + '/no-missing-di/valid-extension-elements.bpmn')
+    },
+    {
+      moddleElement: readModdle(__dirname + '/no-missing-di/valid-signavio.bpmn')
+    },
+    {
+      moddleElement: readModdle(__dirname + '/no-missing-di/valid-complex.bpmn')
+    }
+  ],
+  invalid: [
+    {
+      moddleElement: readModdle(__dirname + '/no-missing-di/invalid-catch-event.bpmn'),
+      report: {
+        id: 'Process',
+        message: 'No bpmndi information found for visual representation of <bpmn:BoundaryEvent> with id=boundaryEvent'
```

Comment:
How do we usually reference elements by `id`, do we have an existing pattern we can reuse here?

Also, I'm wondering if we could shorten the error message. Maybe 

> Missing bpmndi for <bpmn:BoundaryEvent> with ID <boundaryEvent>

Or alternatively, depending on how you name the rule:

> No bpmndi found for ...

Reply:
I shortened the message +1.

I scanned [bpmnlint](https://github.com/bpmn-io/bpmnlint) and [bpmnlint-plugin-camunda](https://github.com/camunda/bpmnlint-plugin-camunda) for patterns for id references in the message itself. 
What we do so far is, we don't give the ID (since the reporter would already output the message for the respective element anyhow).

For `no-bpmndi` however, we IMO cannot do that since the element (although being there) has no visual representation. 

The id referencing pattern (`id=id123`) is afaik what we also use in places outside of bpmnlint (e.g., propertiesPanel). So I sticked with it.
![Screenshot_20210122_092810](https://user-images.githubusercontent.com/42800119/105466176-26c08b00-5c94-11eb-81d0-195d26ff15c5.png)

Reply:
What I'm really wondering: Why don't we log that error on the element that is affected directly? We should do that. This way, we use our built-in facilities to correlate error and element.

Visually speaking, that won't be "drawn" out of the box. We could, however, lift up error messages that cannot be drawn on an element to the first visible element. At least the definitions will be drawn.

Reply:
Considering that we need to adjust that functionality then in the Camunda Modeler plugin (potentially), I completly agree.

Will change accordingly.

Reply:
done. As soon as this PR here is merged, I will follow up with creating issues in the plugin to account for this approach.